hack/generate_chart_manifests.sh

```diff
@@ -3,22 +3,14 @@
 CHART_PATH=../charts/mysql-operator
 CONFIG_PATH=../config
 
-# templates/crds.yaml update
-awk 'FNR==1 && NR!=1 {print "---"}{print}' ${CONFIG_PATH}/crds/*.yaml > ${CHART_PATH}/templates/_crds.yaml
-yq m -d'*' -i ${CHART_PATH}/templates/_crds.yaml chart-metadata.yaml
-yq w -d'*' -i ${CHART_PATH}/templates/_crds.yaml 'metadata.annotations[helm.sh/hook]' crd-install
```

Comment:
the annotations `helm,sh/hook=crd-install` should be kept to be backward compatible.